pyaerocom/web/aerocom_evaluation.py

```diff
@@ -948,6 +967,8 @@ class default will be used (defined in config file)
             self.only_colocation = only_colocation
         if only_json is not None:
             self.only_json = only_json
+
+        self.iface_names = self._check_and_get_iface_names()
```

Comment:
Will not be needed if you make this a property method (see comment below)

Reply:
I'll remove.